src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -541,6 +548,9 @@ private void auditOwnerChange(MediaPatchRequest request, MediaRequestEntity medi
     @Override
     @Transactional
     public MediaHideResponse adminHideOrShowMediaById(Integer mediaId, MediaHideRequest mediaHideRequest) {
+        if (!this.isManualDeletionEnabled()) {
+            throw new DartsApiException(DartsApiException.DartsApiErrorCommon.FEATURE_FLAG_NOT_ENABLED);
+        }
```

Comment:
This need to behave the same as the transcription document hide endpoint, checking the `marked_for_deletion` flag for the associated `reason_id`.

Reply:
Updated